draft-ietf-quic-transport.md

```diff
@@ -2157,8 +2179,22 @@ stream, but is unable to due to the maximum stream ID limit set by its peer (see
 that a new stream was needed, but the stream limit prevented the creation of the
 stream.
 
-The STREAM_ID_BLOCKED frame does not contain a payload.
+The STREAM_ID_BLOCKED frame is as follows:
+
+~~~
+ 0                   1                   2                   3
+ 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+|                        Stream ID (i)                        ...
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+~~~
+
+The STREAM_ID_BLOCKED frame contains a single field.
+
+Stream ID:
 
+: A variable-length integer indicating the stream ID of highest stream that
+could be openend.
```

Comment:
Typo on opened.

I think that we need to be really crisp about this.  Is this the value of MAX_STREAM_ID?  The alternative would be to signal the stream ID of the stream you wanted to create (which would be bad).

Reply:
No, that wasn't my intention. It should be the same value as sent in the MAX_STREAM_ID frame.